src/main/java/frc/robot/subsystems/swerve/SwerveModule.java

```diff
@@ -25,12 +26,15 @@ public class SwerveModule {
     private final WPI_TalonFX driveMotor;
 
     private final CANSparkMax steerMotor;
-    private final SparkMaxAnalogSensor steerEncoder;
+    private final RelativeEncoder steerEncoder;
+    private final SparkMaxAnalogSensor steerAbsoluteEncoder;
     private final SparkMaxPIDController steerPidController;
 
     private final double offsetRads;
 
     private static final double DRIVE_TICKS_TO_METERS = 1.0;
+    private static final double STEER_ROTATIONS_TO_RADIANS = 1.0 / 39.0 * 2 * Math.PI; // 39:1 gear ratio, 1 rotation = 2pi
```

Comment:
jacob sent an update on this constant:

> the gearbox has a ratio of 39:1
> but the pulleys have a ratio of 63:34

Reply:
So it'd be `1.0 / 39.0 * 34.0 / 63.0` for neo rotations to module rotations?

Reply:
i believe so, yes. with parenthesis